src/gluonts/dataset/repository/_gp_copula_2019.py

```diff
@@ -161,4 +161,4 @@ def get_data(dataset_path: Path, ds_info: GPCopulaDataset):
 
 def clean_up_dataset(dataset_path: Path, ds_info: GPCopulaDataset):
     os.remove(dataset_path.parent / f"{ds_info.name}.tar.gz")
-    shutil.rmtree(dataset_path / "metadata")
+    shutil.rmtree(dataset_path / "metadata", ignore_errors=True)
```

Comment:
Is this also needed?

Reply:
yes was getting an error with get_dataset("taxi_30min", regenerate=True) without it... somehow the metadata/ folder gets removed before... I didn't figure it out...